src/Post_Command.php

```diff
@@ -271,6 +275,10 @@ public function create( $args, $assoc_args ) {
 	 *
 	 *     $ wp post update 123 --post_name=something --post_status=draft
 	 *     Success: Updated post 123.
+	 *
+	 *     # Update a post with multiple meta values.
+	 *     $ wp post update 123 --meta_input='{"key1":"value1","key2":"value2"}
```

Comment:
The ' is not balanced here.

it should be :
`--meta_input='{"key1":"value1","key2":"value2"}'`

Reply:
Good catch, thanks!